Demo/Demo/ViewModels/BaseViewModel.swift

```diff
@@ -143,22 +155,13 @@ class BaseViewModel: ObservableObject, PayPalWebCheckoutDelegate, CardDelegate,
         cardClient.approveOrder(request: cardRequest, context: context)
     }
 
-    func checkoutWithNativeClient(orderId: String?) async throws -> NativeCheckoutResult {
+    func checkoutWithNativeClient(orderId: String?) async throws {
         guard let orderId = orderId else {
-
-            return NativeCheckoutResult.error(CoreSDKError(code: 0, domain: "Order Id is null", errorDescription: "Order Id is null"))
+            return
```

Comment:
Should this throw an error?

Reply:
That makes sense! Handling the error in some way, and the way a merchant would, seems like the best route